CHANGELOG.md

```diff
@@ -9,6 +9,10 @@
 **Features**:
 
 - The `std.in` function now supports a list of values (@PrettyWood, #3883)
+- Most standard mathematical functions are now supported: `abs`, `floor`,
+  `ceil`, `pi`, `exp`, `ln`, `log10`, `log`, `sqrt`, `degrees`, `radians`,
+  `cos`, `acos`, `sin`, `asin`, `tan`, `atan` and `pow`.<br/>Note: `pow` can be
```

Comment:
If you have the patience to make a change — we can just put this as a new item, no need to add the HTML tag here. But no big stress.

Reply:
I think `\` can be used instead of `<br/>` here for line continuing.

Reply:
I could use double spaces at the end for a soft line breaking but some linters remove them.
I just added a new item as suggested :)

Reply:
> I could use double spaces at the end for a soft line breaking but some linters remove them.

Pls use `\`

Reply:
Yup sorry done in 1659506